frontend/lib/screens/new_student/local_widget/student_form.dart

```diff
@@ -0,0 +1,209 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:frontend/models/student.dart';
+import 'package:frontend/services/api_service.dart';
+import 'package:frontend/utils/routes_generator.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+
+
+class StudentForm extends StatefulWidget {
+  final Function(bool) changeStep;
+
+  const StudentForm({Key key, this.changeStep}) : super(key: key);
+  @override
+  _StudentFormState createState() => _StudentFormState();
+}
+
+class _StudentFormState extends State<StudentForm> {
+  final _formKey = GlobalKey<FormState>();
+
+  
+  
+  @override
+  void initState() {
+    
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final student = Provider.of<Student>(context);
+    final size = MediaQuery.of(context).size;
+    return SizedBox(
+      width: size.width * .9,
+      height: size.height * .85,
+      child: Container(
+        child: Form(
+          key: _formKey,
+          child: SafeArea(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              mainAxisSize: MainAxisSize.max,
+              children: <Widget>[
+                Container(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(10.0),
+                    color: Colors.white,
+                    boxShadow: [
+                      BoxShadow(
+                        color: Colors.black,
+                        blurRadius: 2.0,
+                        spreadRadius: 0.0,
+                        offset: Offset(2.0, 2.0),
+                      ),
+                    ],
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: 20.0,
+                      vertical: 5.0,
+                    ),
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        _createnameField(student),
+                        _createsurnameField(student),
+                        _createemailField(student),
+                        _createDescriptionField(student),
+                        _createphoneField(student),
```

Comment:
@BassamZ please stick to the design, this way it is not correct: phone field must be before the description field